lib/checker_jobs/notifiers/bugsnag.rb

```diff
@@ -0,0 +1,26 @@
+require "action_mailer"
+
+class CheckerJobs::Notifiers::Bugsnag
+  include CheckerJobs::Notifiers::FormatterHelpers
+
+  class Error < StandardError; end
+
+  def initialize(check, count, entries)
+    @check = check
+    @metadata = {
+      klass: @check.klass,
+      name: @check.name,
+      count: count,
+      entries: entries,
+    }
+  end
+
+  def notify
+    raise Error, "(#{@check.klass}) #{human_check_name} was triggered!"
+  rescue Error => error
+    ::Bugsnag.notify(error, {
+      severity: "warning",
```

Comment:
Should we make this a configuration, the `Logger` notifier does.

Reply:
:+1: I'm planning another PR to configure the way the notification is sent more deeply.